db/models.py

```diff
@@ -50,3 +55,69 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        AUTH_USER_MODEL, on_delete=models.CASCADE, related_name="orders"
+    )
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+    class Meta:
+        ordering = ["-created_at"]
+
+
+class Ticket(models.Model):
+    row = models.IntegerField()
+    seat = models.IntegerField()
+    movie_session = models.ForeignKey(
+        to="db.MovieSession", on_delete=models.CASCADE, related_name="tickets"
```

Comment:
The `to` argument in the ForeignKey should match the actual model name with the correct case. Here, it should be `to=MovieSession` instead of `to="db.MovieSession"`.

Reply:
Agree